### PR TITLE
zh-CN: update get-involved/index.md

### PR DESCRIPTION
**Pull request by ghost**
Add some newly-updated translations.
Related to: https://github.com/nodejs/nodejs.org/pull/3626.

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-cn: Please have a review, thanks!

**Comment by ghost** [comment]
@laosb: Can you help to have a review of it? Thanks!

File: locale/zh-cn/get-involved/index.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ layout: contribute.hbs
 ## 社区讨论
 
 * [GitHub 议题清单](https://github.com/nodejs/node/issues)是讨论 Node.js 核心功能的好地方。
-* 关于 Node.js 开发的实时对话，请使用 [IRC 客户端](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) 或 [freenode 网页聊天室](https://webchat.freenode.net/#node.js)转到 `irc.freenode.net` 中的 `#node.js` 頻道。
+* 想实时聊聊关于 Node.js 开发方面的内容，可以使用以下之一的平台：
+  * 使用 IRC，请使用 [IRC 客户端](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients)，在 `#node.js` 频道中转至 `irc.freenode.net`，或者在您的浏览器中使用 [freenode 网页版聊天](https://webchat.freenode.net/#node.js) 直接连接即可。
```
Comment (laosb) [inline locale/zh-cn/get-involved/index.md]: 是`irc.freenode.net`上的`#node.js`頻道吧？

**Review by laosb — CHANGES_REQUESTED** [review]

File: locale/zh-cn/get-involved/index.md (hunk 1 above)
Comment (MaledongGit) [reply locale/zh-cn/get-involved/index.md:12]: 是。不过英语原文是：For IRC, go to `irc.freenode.net` in the `#node.js` channel with an [IRC client]....，我只能照直翻译了。

Comment (MaledongGit) [reply locale/zh-cn/get-involved/index.md]: 我现在改过来了，它那个英文版你怎么理解？我觉得有问题耶？

Comment (MaledongGit) [reply locale/zh-cn/get-involved/index.md]: 我现在改过来了，它那个英文版你怎么理解？我觉得有问题耶？@laosb

Comment (ghost) [reply locale/zh-cn/get-involved/index.md]: Thanks, fixed ;0

**Pull request merged**